config/prod/prow/k8s-testgrid/k8s-testgrid.yaml

```diff
@@ -0,0 +1,74 @@
+# Dashboards need to be specified here to be created on TestGrid
```

Comment:
It looks this file is auto-generated, can we add a comment here and state it like how we are handling other auto-generated files?

Reply:
Done.